src/checkout.c

```diff
@@ -2544,6 +2544,17 @@ static int checkout_data_init(
 #define CHECKOUT_INDEX_DONT_WRITE_MASK \
 	(GIT_CHECKOUT_DONT_UPDATE_INDEX | GIT_CHECKOUT_DONT_WRITE_INDEX)
 
+GIT_INLINE(void) setup_pathspecs(
+	git_iterator_options *iter_opts,
+	const git_checkout_options *checkout_opts)
+{
+	if (checkout_opts &&
+		(checkout_opts->checkout_strategy & GIT_CHECKOUT_DISABLE_PATHSPEC_MATCH)) {
```

Comment:
Nit: indentation. Continuation lines start with the opening brace :)

Reply:
Show me what you think our continuation standard looks like.  😛 

Are you saying:

```
if (checkout_opts &&
   (checkout_opts->...
```